lib/streamlit/elements/plotly_chart.py

```diff
@@ -73,7 +84,7 @@ def plotly_chart(
         figure_or_data: FigureOrData,
         use_container_width: bool = False,
         sharing: SharingMode = "streamlit",
-        theme: Union[None, Literal["streamlit"]] = None,
+        theme: Union[None, Literal["streamlit"]] = "streamlit",
```

Comment:
We probably need to put this to `None` again since we aren't activating this as default for now?